README.md

```diff
@@ -141,27 +141,38 @@ can use the following:
   description = "A very basic flake";
 
   inputs.nixvim.url = "github:nix-community/nixvim";
-  inputs.flake-utils.url = "github:numtide/flake-utils";
 
   outputs = {
     self,
-    nixpkgs,
     nixvim,
-    flake-utils,
-  }: let
+    flake-parts,
```

Comment:
This is missing the flake-parts input I think?

Reply:
Actually, you don't need it thanks to the magic flake registry: https://github.com/NixOS/flake-registry
The examples provided in the flake-parts repo itself don't put it in the inputs.

Reply:
Well I don't think it's good to rely on the flake registry, because we don't control the versions. 
![image](https://github.com/nix-community/nixvim/assets/5623227/c9e5cd01-3e86-47b9-a040-f2cc53813d5a)
I don't agree that the example does not provide it.

Reply:
I think it's different for a "library" where we need to tightly control the versions of our inputs to avoid random breakage on a machine & a "end-user" flake, where taking what's on the machine is ok.